wolfssl/wolfcrypt/asn.h

```diff
@@ -2542,6 +2542,10 @@ WOLFSSL_LOCAL int SetAsymKeyDerPublic(const byte* pubKey, word32 pubKeyLen,
 WOLFSSL_LOCAL int DecodeAsymKeyPublic_Assign(const byte* input,
     word32* inOutIdx, word32 inSz, const byte** pubKey, word32* pubKeyLen,
     int keyType);
+WOLFSSL_LOCAL int DecodeAsymKeyPublic_Assign_ex(const byte* input,
```

Comment:
Since `DecodeAsymKeyPublic_Assign` is a local function it is fine with me to change all uses of it... Vs asking another _ex.

Reply:
Ok, made the change in [2207791](https://github.com/wolfSSL/wolfssl/pull/8177/commits/2207791aab0a153ae13e250118c549817a33de7f). If @SparkiDev insists on the _ex function as he originally suggested we can just revert that commit.